addons/udes_get_info/tests/test_stock_quant.py

```diff
@@ -30,6 +30,7 @@ def setUpClass(cls):  # noqa: D102
                 "package_id",
                 "product_id",
                 "quantity",
+                "lot_id",
```

Comment:
Q: The change in models.py setting the `_info = False` when there is no result, is that needed to get the test to run? or is it a quality of life improvement. As it seems to me that the test would run fine just with this change if the quant had a lot_id.

Reply:
The change in models `_info = False` fixed the failing test as the `test_retrieves_expected_fields_from_related_model` which was the failing test dont rely on stock quant expected_keys. 

But we need to change expected keys as well after the change made to _info = False as it was failing the test_stock_quant which makes sense as we have added a new field on Quant._get_info_field_names

Reply:
Spoke in a call, I am happy with the change :smile: